content/en/docs/contribute/generate-ref-docs/kubernetes-api.md

```diff
@@ -175,15 +162,51 @@ git status
 
 The output shows the modified files:
 
+```
+static/docs/reference/generated/kubernetes-api/v1.15/css/bootstrap.min.css
+static/docs/reference/generated/kubernetes-api/v1.15/css/font-awesome.min.css
+static/docs/reference/generated/kubernetes-api/v1.15/css/stylesheet.css
+static/docs/reference/generated/kubernetes-api/v1.15/fonts/FontAwesome.otf
+static/docs/reference/generated/kubernetes-api/v1.15/fonts/fontawesome-webfont.eot
+static/docs/reference/generated/kubernetes-api/v1.15/fonts/fontawesome-webfont.svg
+static/docs/reference/generated/kubernetes-api/v1.15/fonts/fontawesome-webfont.ttf
+static/docs/reference/generated/kubernetes-api/v1.15/fonts/fontawesome-webfont.woff
+static/docs/reference/generated/kubernetes-api/v1.15/fonts/fontawesome-webfont.woff2
+static/docs/reference/generated/kubernetes-api/v1.15/index.html
+static/docs/reference/generated/kubernetes-api/v1.15/jquery.scrollTo.min.js
+static/docs/reference/generated/kubernetes-api/v1.15/navData.js
+static/docs/reference/generated/kubernetes-api/v1.15/scroll.js
+```
+
+## Updating the API reference index pages
+
+
+* Open `<web-base>/content/en/docs/reference/kubernetes-api/index.md` for editing, and update the API reference 
+  version number. For example:
+
+    ```
+    ---
+    title: v1.15
+    ---
+
+    [Kubernetes API v1.15](/docs/reference/generated/kubernetes-api/v1.15/)
+    ```
+
+* Open `<web-base>/content/en/docs/reference/_index.md` for editing, and add a
+   new link for the latest API reference. Remove the oldest API reference version.
+   There should be five links to the most recent API references.
+
+
+## Locally test the documentation
+
+Build the Kubernetes documentation and verify the local preview.
+
 ```shell
-On branch master
-...
-   modified:   docs/reference/generated/kubernetes-api/v1.9/index.html
+cd <web-base>
+make docker-serve
```

Comment:
Maybe mention that the reader should look at http://localhost:1313/ and review the changes?
It's also OK to infer that if they got this far they already know to do that.